addons/l10n_pe_website_sale/static/src/js/website_sale.js

```diff
@@ -40,17 +39,20 @@ WebsiteSale.include({
             }
         });
     },
-    _onChangeState: function () {
-        if (this.isPeruvianCompany) {
-            if (this.elementState.value === "" && this.elemenCountry.value !== '') {
-                this.elementState.options[1].selected = true;
+    _onChangeState: function (ev) {
+        return this._super.apply(this, arguments).then(() => {
+            let selectedCountry = this.elemenCountry.options[this.elemenCountry.selectedIndex].getAttribute("code");
+            if (this.isPeruvianCompany && selectedCountry == "PE") {
```

Comment:
we prefer to use the strict equality, even if I don't see any case where `selectedCountry == "PE"` and `selectedCountry === "PE"` would give different outcomes :)